_layouts/post.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+---
+
+<div class="container post-content">
```

Comment:
This looks weird to me, compare

![image](https://user-images.githubusercontent.com/1552237/68640095-a061d280-0541-11ea-8dce-f74bd8840321.png)
and 
![image](https://user-images.githubusercontent.com/1552237/68640103-a788e080-0541-11ea-8b26-aa56d3c465ca.png)

The second seems more readable to me in every respect:

- header not truncated
- font readable for long texts, standard font and not 'landing page/marketing/short-text font'
- wider body text
- can see date and author

Lets keep what is working and make this closer to the original IMO

Reply:
agree

Reply:
I improved it. Let me know how it looks.

Reply:
looks better to me, not 100% sure about the margins/spacing

![image](https://user-images.githubusercontent.com/1552237/68937338-277ba880-07d7-11ea-9dc5-1bebdf6abee4.png)

but don't know exactly how it should be improved. @j-zerah and @decanus wdyt?

Another thing I just realized - it'd be nice if we had a static page index for posts, as the screen real estate looks quite different for all posts under vac.dev:

![image](https://user-images.githubusercontent.com/1552237/68937412-4da14880-07d7-11ea-9d9c-2f3690e4a7a6.png)


and #research-log http://localhost:4000/#research-log:
![image](https://user-images.githubusercontent.com/1552237/68937437-585bdd80-07d7-11ea-892a-578b5a2efdd3.png)

What about having research log link to `/posts` which has an index more similar to the one under vac.dev?